### DISCUSSION TITLE
ES file explorer is not in the infotainment apps menu after plug in for installation and reboot

### DISCUSSION THREAD

**Discussion by Kevin-lzw** [xeon1989/Proton-X50-APK-Installer#11] [Q&A]
Hi, i have done everything up to step 5, the ES file explorer could not be found in the apps menu.

i saw the infotaiment system rebooted a few seconds later after i plugged in the USB, and the screen showed optimizing apps which i assumed was installation, then the infotaiment system started as usual. When i clicked the tab button at the bottom of the screen to change to apps menu, i swiped to 2nd page and the ES file explorer was not there.

I think i did it right according to the steps provided but i could not find any issue causing this. Can you please help me to fix this? Thank you very much in advance, and i really appreciate for your help.

**Comment by xeon1989** [comment]
I believe this is caused by the ES you are not compatible with the Android version the HU running on. Please try redownloading ES and make sure the version you downloaded supports Android 5.1.

**Reply by Kevin-lzw** [reply]
Hi, thanks for the reply, actually I clicked the link you provided and downloaded the ES file explorer from the apkpure. So the download link cannot be used anymore as the apk version might be outdated?

Thanks again.
